life_expectancy/tests/conftest.py

```diff
@@ -24,6 +24,16 @@ def pt_life_expectancy_expected() -> pd.DataFrame:
 
 
 @pytest.fixture(scope="session")
-def input_dataframe_test() -> pd.DataFrame:
-    """Fixture to load the expected output of the cleaning script"""
+def input_dataframe_test_csv() -> pd.DataFrame:
+    """Fixture to load the input of the cleaning script for a csv file"""
     return pd.read_csv(FIXTURES_DIR / "eu_life_expectancy_raw_testing.tsv", sep="\t")
+
+@pytest.fixture(scope="session")
+def input_dataframe_test_json() -> pd.DataFrame:
+    """Fixture to load the input of the cleaning script for a json file"""
+    return pd.read_json(FIXTURES_DIR / "eurostat_life_expect.json")
+
+@pytest.fixture(scope="session")
+def output_expected_country_list() -> pd.DataFrame:
+    """Fixture to load the input of the cleaning script for a json file"""
```

Comment:
Did not understand this function, since the comments are not explanatory.

Reply:
esta é a função que devolve em forma de lista os paises válidos da lista total de países